### DISCUSSION TITLE
AsyncConnectionPool.__init__()  Error

### DISCUSSION THREAD

**Discussion by jschintz-windriver** [encode/httpcore#843] [Potential Issue]
The following error popped up this morning on ubuntu.  Has anyone else seen this?

`Encountered unexpected TypeError: "AsyncConnectionPool.__init__() got an unexpected keyword argument 'socket_options'"`

**Comment by lovelydinosaur** [comment]
* What versions of `httpx` and `httpcore` are you using?
* Does the error disappear if you update to the latest versions?
